input/network_config.yml

```diff
@@ -33,3 +33,13 @@ mlnx_ofed_version: 5.4-2.4.1.3
 # Mandatory variable
 # Default value: true
 mlnx_ofed_add_kernel_support: true
+
+# Absolute path to local copy of .tgz file containing DOCA package.
+# The package can be downloaded from https://developer.nvidia.com/networking/doca/
+# Optional variable.
+nvidia_doca_offline_path: ""
```

Comment:
Question: during testing I see mix between `nvidia_doca_path` and `nvidia_doca_offline_path` need to review this is more details